src/test/java/uk/gov/companieshouse/chs/gov/uk/notify/integration/api/restapi/SenderRestApiIntegrationTest.java

```diff
@@ -667,45 +667,56 @@ void sendLetterWithIncorrectlyNamedMonth(CapturedOutput log) throws Exception {
         verifyNoLetterResponsesAreStored();
     }
 
-    @ParameterizedTest(name = "Send letter with economy postage for {0}")
-    @CsvSource({ "CSIDVDEFLET,v1.0,send-csidvdeflet-request",
-            "IDVPSCDEFAULT,v1.0,send-idvpscdefault-request" })
-    void sendLetterWithEconomyPostage(String letterType, String templateId, String reference, CapturedOutput log)
+    @ParameterizedTest(name = "Send letter with economy postage for {0} {1}")
+    @CsvSource({ "CSIDVDEFLET,v1.0,4221479,send-csidvdeflet-request",
```

Comment:
There are magic numbers and magic strings in this test. If we replace them with constants the code will be more meaningful.